entity-management-web/src/main/java/eu/europeana/entitymanagement/config/ValidatorConfig.java

```diff
@@ -0,0 +1,52 @@
+package eu.europeana.entitymanagement.config;
+
+import static eu.europeana.entitymanagement.common.config.AppConfigConstants.BEAN_EM_LANGUAGE_CODES;
+import static eu.europeana.entitymanagement.common.config.AppConfigConstants.BEAN_EM_VALIDATOR_FACTORY;
+import static eu.europeana.entitymanagement.common.config.AppConfigConstants.BEAN_XML_MAPPER;
+
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import eu.europeana.entitymanagement.common.config.EntityManagementConfiguration;
+import eu.europeana.entitymanagement.common.config.LanguageCodes;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.stream.Collectors;
+import javax.validation.ValidatorFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
+
+@Component
+public class ValidatorConfig {
```

Comment:
Moved `ValidatorFactory` config to a separate class so EntityFieldsValidatorTest can explicitly load it (instead of the entire AppConfig).